### PR TITLE
Footer component with ContactPerson component

### PR DESCRIPTION
**Pull request by erlenste**
add footer for new levelup page. contains a ContactPerson component that is tested for 2 contact persons.
image

### CODE REVIEW

File: src/components/Footer/Footer.tsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+import ContactPerson from "../ContactPerson";
+import { FooterTitle, PageFooter, PersonContaier } from "./styled";
+
+function Footer(props : any) {
+  var {pageText} = props;
+  return(
+    <PageFooter>
+      <FooterTitle>{pageText}</FooterTitle>
+      <PersonContaier>  
+        <ContactPerson image="https://www.iconninja.com/files/592/204/968/communication-male-cyber-avatar-man-person-crime-human-profile-head-anonymous-hacker-creative-icon.svg" name="Erlend Stenlund" email="erlend.stenlund@itera.com"/>
```
Comment (LERoise) [inline src/components/Footer/Footer.tsx:11]: Are you sure you want to add yourself as a contact person ?

Comment (erlenste) [reply src/components/Footer/Footer.tsx:11]: @LERoise I'm definitely not sure about that 😅
Would you like to be a contact person?

Comment (anhtin) [reply src/components/Footer/Footer.tsx:11]: Do we need any specific contact person? It'd be better to just link to the mad-learning channel on Slack

File: src/components/Footer/styled.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import { usingTypography } from 'src/hooks/theme';
+import styled from 'styled-components';
+
+export const PageFooter = styled.footer`
+  width: 100%;
+  height: 300px;
+  background-color: lightgray;
```
Comment (anhtin) [inline src/components/Footer/styled.ts:7]: Color should be added to the color palette somehow, if it's not there already 😝

File: src/components/Footer/styled.ts [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+`;
+
+export const FooterTitle = styled.p`
+  margin: 0 auto;
+  padding-top: 60px;
```
Comment (anhtin) [inline src/components/Footer/styled.ts:12]: Might want to use topography here too

**Pull request merged**